errors.go

```diff
@@ -77,7 +81,9 @@ func HandleExitCoder(err error) {
 	}
 
 	if exitErr, ok := err.(ExitCoder); ok {
-		if err.Error() != "" {
+		if _, ok := exitErr.(ErrorFormatter); ok {
```

Comment:
That is a bit weird, I'm not sure why that check is there. Let me see if I can track down when it was added.